source/production/F0.Minesweeper.Components/Abstractions/ICell.cs

```diff
@@ -0,0 +1,12 @@
+using F0.Minesweeper.Components.Abstractions.Enums;
+using F0.Minesweeper.Logic.Abstractions;
+
+namespace F0.Minesweeper.Components.Abstractions
+{
+    public interface ICell 
+    {
+        Location? Location { get; set; }
```

Comment:
nullable Location - when does a Cell have no location?

Reply:
That's because the `Location` is injected as a `Parameter` to the `Cell` component. Hopefully this issue will be fixed in the next version of Blazor